daml-lf/transaction/src/main/scala/com/daml/lf/TransactionNodesStatistics.scala

```diff
@@ -54,11 +54,17 @@ final case class TransactionNodesStatistics(
   def nodes: Int = actions + rollbacks
 }
 
+final case class TransactionNodesStatistics(
```

Comment:
the name here seems a bit fishy. The fact that `Stats` and `Statistics` are not the same thing is fairly confusing. Not really sure what a better name is though :disappointed: